packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts

```diff
@@ -229,7 +230,7 @@ function getSuggestionsForFieldNames(
         sortText: String(index) + field.name,
         label: field.name,
         detail: String(field.type),
-        documentation: field.description,
+        documentation: field.description || undefined,
```

Comment:
we have a `Maybe` that might be more consistent to use here?

Reply:
Yes but I didn't find a better approach ...

`CompletionItem` is defined in `vscode-languageserver-types`

```ts
CompletionItem.documentation?: string | MarkupContent | undefined
```

and `field.description` is defined in `graphql`

```ts
GraphQLEnumValue.description: Maybe<string>
```

Reply:
`?? undefined` seems to be better than `|| undefined`...

@acao do you have some advice here?

Reply:
@stonexer agreed! this rewrite predatates nullish coalescing, so now that it's available, that would be a good addition

Reply:
oh! yes then `| undefined` is fine. i thought we had a `type Maybe<T> = T | undefined` in our types